actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -619,12 +620,8 @@ def apply_form_for_options!(object, options) # :nodoc:
       #   When set to <tt>true</tt>, the form is submitted via standard HTTP.
       #   When set to <tt>false</tt>, the form is submitted as a "remote form", which
       #   is handled by Rails UJS as an XHR. When unspecified, the behavior is derived
```

Comment:
- Should we remove the UJs reference since it's not included in new apps?
- Does this functionality work the same without UJs?

Reply:
I haven't tested UJS, but assuming that we haven't changed any behavior I think it's fine?